dev/com.ibm.ws.security.authorization.jacc.ejb/bnd.bnd

```diff
@@ -16,6 +16,15 @@ Bundle-SymbolicName: com.ibm.ws.security.authorization.jacc.ejb
 Bundle-Description: WAS Security EJB support for JACC Service, version=${bVersion}
 Bundle-ActivationPolicy: lazy
 
+#
+# Produce a JakartaEE compliant JAR.
+#
+jakartaeeMe: true
+jakartaee.transform.jar.name=io.openliberty.security.authorization.jacc.internal.ejb.jar
+JakartaEE-Bundle-Name: EJB support for Jakarta Authorization 2.0
+JakartaEE-Bundle-Symbolic-Name: io.openliberty.security.authorization.jacc.internal.ejb
+JakartaEE-Bundle-Description: EJB support for Jakarta Authorization 2.0
+JakartaEE-Bundle-Version: 2.0
 
 WS-TraceGroup: Authorization
 
```

Comment:
Need to look at the generated manifest to see if the JAXRPC is added to the manifest for the Jakarta EE 9 version of the bundle.  If it is, then you may need to adjust the imports to make it optional possibly.

Reply:
With the current version of the EJBSecurityValidatorImpl class, no JAX RPC classes are either imported or exported from the bundle. I will check when I make the changes suggested in the review findings for that class.

Reply:
To be clear you are looking for the package, not the class.

Reply:
Yes. Understood. I should have been clearer.

However; after these latest changes I see the `javax.xml.rpc.handler;version="[1.1,2)",` package imported. I am assuming removing reflection resulted in this package being picked up.

Reply:
You can put in the reflection or make it optional I believe.

Reply:
Right now none of this code will be called because there is no io.openliberty.ejbCore-2.0 feature for the auto feature to find and cause the auto feature to be used.

Reply:
I believe that all the necessary dependent bundles and features exist to create a ejbCore-2.0 private feature like you are depending on.  It is best to hear what @tkburroughs has to say about ejbCore status and if you should make a ejbCore-2.0 as part of this pull request.  None of your ejb tests will currently do jakarta EJB 4.0 since that isn't part of the replacement action yet for JakartaEE 9 and there is no ejbLite-4.0 or ejb-4.0 feature yet.

Reply:
Tracy has a PR for that: https://github.com/OpenLiberty/open-liberty/pull/12935

Reply:
Yep, ejbCore-2.0 is nearly ready (pending review comments)... so will be there soon.

Reply:
I have made resolution of the RPC package optional.